Upgrade Kaptain for Konvoy 1.x (#4123)
mesosphere/dcos-docs-site: pages/dkp/kaptain/1.3.0/upgrade/index.md

          enabled: true
    ```

- Ensure the Kaptain addon repository is present in your Konvoy `cluster.yaml`:
  
  ```yaml
      - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
        configVersion: stable-1.20-1.4.0
        addonsList:
          - name: knative
            enabled: true
  ```
  
## Upgrade Kaptain 

> 1. Perform the upgrade:

Reviewer: ```suggestion
Run the upgrade according to your Konvoy version:
```

### For Konvoy 1.x: 

> Add the following properties to your `params.yaml` file: 

Reviewer: ```suggestion
1.   Add these properties to your `params.yaml` file:
```

  ```bash
  dkpPlatformVersion: "1" 
  installMinioOperator: "true"
  ```

> And upgrade Kaptain: 

Reviewer: ```suggestion
1.   And the Kaptain upgrade : 
```

  ```bash
  kubectl kudo upgrade --instance kaptain --namespace kubeflow ./kubeflow-1.4.0_1.3.0.tgz -P params.yaml
  ```
  
### For Konvoy 2.x:

> Upgrade Kaptain:

Reviewer: ```suggestion
1.  Upgrade Kaptain:
```

  ```bash
  kubectl kudo upgrade --instance kaptain --namespace kubeflow ./kubeflow-1.4.0_1.3.0.tgz
  ```

> 2. Monitor the upgrade process by running:

Reviewer: ```suggestion
1.  Monitor the upgrade process by running:
```

Reviewer: Markdown dictates DO NOT use sequential numbers, always use "1.  " (note the two blank spaces)

  
  ```bash
  kubectl kudo plan status --instance kaptain --namespace kubeflow
  ```

3. After the upgrade completes, log in to Kaptain.

> 1. Discover the cluster endpoint and copy it to the clipboard.

Reviewer: ```suggestion
1.  Locate the cluster endpoint and copy it to your clipboard.
```

  
  If you are running Kaptain _on-premises_, use this command:

  ```bash
  kf_uri=$(kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].ip}") && echo "https://${kf_uri}"
  ```

>   If you are running Kaptain on _AWS_, use this command:

Reviewer: ```suggestion
  **OR** If you are running Kaptain on _AWS_, use this command:
```

  
  ```bash
  kf_uri=$(kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].hostname}") && echo "https://${kf_uri}"
  ```

> 5. Get the login credentials from Konvoy to authenticate:

Reviewer: ```suggestion
1.  Obtain the login credentials from Konvoy to authenticate:
```
